src/makeCalculator.js

```diff
@@ -4,7 +4,37 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    add(a, b) {
+      return a + b;
```

Comment:
[CODE STYLE] - Use arithmetic operators with assignment operator `=` if you want to add something to existing value. Instead of `return a + b;`, you can use `a += b; return a;` to follow the good example provided in the checklist.